config/staging/prow/core/plugins.yaml

```diff
@@ -121,6 +126,32 @@ plugins:
   - welcome
   - wip
   - yuks
+  knative-prow-robot/test-infra:
```

Comment:
Will it work to have two `knative-prow-robot/test-infra`s in this file?

Reply:
Actually, I think at least it won't fail prow, see https://github.com/knative/test-infra/blob/350178ab2a0e4780bc9d039cf7a72b3d4ce183b9/config/staging/prow/core/config.yaml#L136. 

It might override the wrong way, but since this only affects staging prow, and staging prow is already covered by auto config updater, we should be fine